packages/design-system/src/components/N8nPagination/Pagination.vue

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+import { Pagination as ElPagination } from 'element-ui';
+
+export default {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access
+	props: (ElPagination as any).props,
+	components: {
+		ElPagination,
+	},
+};
+</script>
+
+<template>
+	<div>
+		<el-pagination
```

Comment:
would be helpful to replace this in the NDV as well..

Reply:
Yes, definitely. I'll add a tech-debt task. The tags list as well.